.travis.yml

```diff
@@ -21,12 +18,11 @@ env:
 matrix:
   exclude:
     - env: HANDLER="ox"
-      rvm: jruby-1.7
+      rvm: jruby-9.2.6.0
```

Comment:
I think you can safely use `jruby-9` instead and travis will figure out what the current release is for you

Reply:
I have not been successful letting Travis CI's `rvm` pick a Ruby which does that for me.

In `rvm`, the information _does_ exist. https://github.com/rvm/rvm/blob/master/config/known#L19

Reply:
Interesting, for some reason feedjira doesn't seem to mind.
https://github.com/feedjira/feedjira/blob/master/.travis.yml#L9

Either way, dropping jruby-1.7 seems like a great idea! 💯